.github/workflows/test.yml

```diff
@@ -1,54 +0,0 @@
-name: Test
-on: [pull_request]
-jobs:
```

Comment:
I think every other file in this directory ends in .yml, right? Can we keep it consistent?